### PR TITLE
2020.09 changelog

### PR DESCRIPTION
**Pull request by tobiasd**
Work in progress for the CHANGELOG file of the 2020.09 release.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -14,6 +14,7 @@ Version 2020.09 (unreleased)
     Enhanced the performance of the daemon [annando]
     Enhanced the communication with the directory servers [annando]
     Enhaceed the resharing of items [annando]
+    Enhanced sample lighttpd and nginx configs [MrTetovan, tobiasd]
```
Comment (MrPetovan) [inline CHANGELOG]: 😂
```suggestion
    Enhanced sample lighttpd and nginx configs [MrPetovan, tobiasd]
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: CHANGELOG (hunk 1 above)
Comment (annando) [reply CHANGELOG]: Monsieur la tête aux vin. :grin:

Comment (tobiasd) [reply CHANGELOG]: One typo removed, one ypot added...

File: CHANGELOG [hunk 2]
```diff
@@ -46,6 +48,10 @@ Version 2020.09 (unreleased)
     8914, 8928, 8929, 8935, 8940, 8941, 8967, 8989, 8993, 8994, 8995,
     8997, 8999, 9000, 9004, 9013, 9015, 9051, 9064, 9065, 9072, 9107
 
+Version 2020.07-1 (2020-09-08)
+  Friendica Core
+    Fixed a problem that leakted sensitive information [Roger Meyer, MrPetovan]
```
Comment (annando) [inline CHANGELOG]: "Leaked"

**Review by MrPetovan — APPROVED** [review]

File: CHANGELOG [hunk 3]
```diff
@@ -25,6 +25,7 @@ Version 2020.09 (unreleased)
     Fixed missing HTML encoding [MrPetovan]
     Fixed a layout problem with the frio composer for new postings [MrPetovan]
     Fixed some composer notices [nupplaphil]
+    Fixed a problem while displaying previews for empty feets [annando]
```
Comment (annando) [inline CHANGELOG]: "Fixed a problem for empty preview data when importing feed posts"

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**